### PR TITLE
CC-29892: Fixed payment methods for checkout-data endpoint

### PR DESCRIPTION
**Pull request by ievgen-malykhin**
#### Overview

- Ticket: https://spryker.atlassian.net/browse/CC-29892

###### Optional

- Core PR: None

###### Change log

{Relevant changes for project level go here. Those will be copied to the release log for suite if necessary.}

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -308,7 +308,6 @@
         "allow-plugins": {
             "dealerdirect/phpcodesniffer-composer-installer": true,
             "composer/package-versions-deprecated": true,
-            "sllh/composer-versions-check": true,
```
Comment (dmiseev) [inline composer.json:311]: Please revert the changes.

**Comment by ievgen-malykhin** [comment]
should be recreated.
